lib/containers/urls.pm

```diff
@@ -342,4 +298,26 @@ sub get_3rd_party_images {
     return (\@images);
 }
 
+# Return url of specified image as a string. Die when nothing to return.
```

Comment:
```suggestion
# Return URI of the container image to be tested. This can be either configured via the CONTAINER_IMAGE_TO_TEST variable or it will try to determinte the same image as the running OS. Dies if the image could not be determined
```

WDYT?

Reply:
There are 3 cases:

- Provide `CONTAINER_IMAGE_TO_TEST` variable with URI
- Give it as input parameter `$args{version}` which is what we do to test our big matrix.
- Determine the same image as the running OS.

Reply:
```
# Return URI of the container image to be tested. This will be:
# - Configured by the CONTAINER_IMAGE_TO_TEST variable.
# - Configured by the distri, version and arch parameter.
# - Determined by the running OS.
# - Terminated otherwise.
```

Reply:
Please re-check.